BrainPortal/app/models/message.rb

```diff
@@ -74,7 +75,7 @@ def self.send_message(destination, options = {})
     expiry       = options[:expiry]        || options["expiry"]
     critical     = options[:critical]      || options["critical"]      || false
     send_email   = options[:send_email]    || options["send_email"]    || false
-    group_id     = options[:group_id]      || options["group_id"]
+    group_id     = options[:group_id]      || options["group_id"]      # For invitation messages only.
```

Comment:
So we should rename the variable and the key in the options hash to 'invitation_group_id' and make sure that the change is applied in places where this is called (which should only be when making Invitation objects)